core/domain/exp_jobs_one_off.py

```diff
@@ -434,7 +435,7 @@ def map(item):
                 yield (
                     'ERROR with exp_id %s version %s' % (
                         item.id, exploration.version),
-                    str(e))
+                    traceback.format_exc())
```

Comment:
I tested this in the console and this returns a string of stack trace. So we would be able to see this in logs right?

Reply:
I think so. Just to check, does it include what would have been in `str(e)` as well?

(You can test this, btw. Just run it locally and look at the output on the admin page. Better to do that before running in prod.)

Reply:
Yes it does. Tested on Oppia (raised exception in try block) and output is:
```
[u'ERROR with exp_id 0 version 1', [u'Traceback (most recent call last):\n File "/home/prasanna/opensoure/oppia/core/domain/exp_jobs_one_off.py", line 434, in map\n raise Exception(\'My own Exception\')\nException: My own Exception\n']]
```
So it does work.